[2021-08-12 10:34:54,034] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2021-01-06T00:00:00+00:00 [queued]>
[2021-08-12 10:34:54,038] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2021-01-06T00:00:00+00:00 [queued]>
[2021-08-12 10:34:54,038] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:34:54,038] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 10:34:54,038] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:34:54,045] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_3> on 2021-01-06T00:00:00+00:00
[2021-08-12 10:34:54,047] {standard_task_runner.py:52} INFO - Started process 130370 to run task
[2021-08-12 10:34:54,051] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', '2021-01-06T00:00:00+00:00', '--job-id', '49', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/subdag_example.py', '--cfg-path', '/tmp/tmpa5w1fxet', '--error-file', '/tmp/tmpugkcgglz']
[2021-08-12 10:34:54,052] {standard_task_runner.py:77} INFO - Job 49: Subtask task_3
[2021-08-12 10:34:54,073] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_3 2021-01-06T00:00:00+00:00 [running]> on host adam
[2021-08-12 10:34:54,096] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2021-01-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-06T00:00:00+00:00
[2021-08-12 10:34:54,097] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 10:34:54,097] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo task 3']
[2021-08-12 10:34:54,101] {subprocess.py:74} INFO - Output:
[2021-08-12 10:34:54,102] {subprocess.py:78} INFO - task 3
[2021-08-12 10:34:54,102] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 10:34:54,113] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20210106T000000, start_date=20210812T083454, end_date=20210812T083454
[2021-08-12 10:34:54,123] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-12 10:34:54,142] {local_task_job.py:149} INFO - Task exited with return code 0
